tests/test_modeling_common.py

```diff
@@ -3385,6 +3386,9 @@ def test_eager_matches_sdpa_inference(self, torch_dtype: str):
         if torch_device == "cpu" and torch_dtype == "float16":
             self.skipTest("float16 not supported on cpu")
 
+        if torch_dtype == "bfloat16" and not is_torch_bf16_available_on_device(torch_device):
```

Comment:
It would be great to update the check above too to use `is_torch_fp16_available_on_device`